application/frontend/controllers/SiteController.php

```diff
@@ -40,14 +40,23 @@ public function actionStatus()
             Yii::error('DB connection problem: ' . $e->getMessage());
             throw new Http500('DB connection problem.', $e->getCode());
         }
-        
+
+
         try {
             $emailer = $webApp->get('emailer');
         } catch (Exception $e) {
             Yii::error('Emailer config problem: ' . $e->getMessage());
             throw new Http500('Emailer config problem.');
         }
-        
+
+        // Checking the email service status causes too many error emails to be sent out to the dev team
+        // $this->checkEmailServiceStatus($emailer);
+
+        Yii::$app->response->statusCode = 204;
```

Comment:
Had to look up 204, but that seems okay.
https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html

Reply:
Also, this is the better placement for this line of code, since all the other checks are try/catch. It's weird that a private function sets the return value for this function.